integration/dubbo-alibaba/src/main/java/io/seata/integration/dubbo/alibaba/AlibabaDubboTransactionPropagationFilter.java

```diff
@@ -43,45 +45,56 @@ public Result invoke(Invoker<?> invoker, Invocation invocation) throws RpcExcept
             return invoker.invoke(invocation);
         }
         String xid = RootContext.getXID();
-        String xidInterceptorType = RootContext.getXIDInterceptorType();
+        String branchType = RootContext.getBranchType();
 
         String rpcXid = getRpcXid();
-        String rpcXidInterceptorType = RpcContext.getContext().getAttachment(RootContext.KEY_XID_INTERCEPTOR_TYPE);
+        String rpcBranchType = RpcContext.getContext().getAttachment(RootContext.KEY_BRANCH_TYPE);
         if (LOGGER.isDebugEnabled()) {
             LOGGER.debug("xid in RootContext[{}] xid in RpcContext[{}]", xid, rpcXid);
         }
         boolean bind = false;
         if (xid != null) {
             RpcContext.getContext().setAttachment(RootContext.KEY_XID, xid);
-            RpcContext.getContext().setAttachment(RootContext.KEY_XID_INTERCEPTOR_TYPE, xidInterceptorType);
+            RpcContext.getContext().setAttachment(RootContext.KEY_BRANCH_TYPE, branchType);
         } else {
             if (rpcXid != null) {
                 RootContext.bind(rpcXid);
-                RootContext.bindInterceptorType(rpcXidInterceptorType);
+                if (StringUtils.equals(BranchType.TCC.name(), rpcBranchType)) {
+                    RootContext.bindBranchType(BranchType.TCC);
```

Comment:
Saga branch never set ?

Reply:
I have discussed with the author of saga. This pr is only responsible for the branchType judgment logic in ExecuteTemplate and the TCC branchType passing in Filter. The specific Saga branchtype assignment and other logic will be provided by the feature saga related pr

Reply:
this https://github.com/seata/seata/pull/2551, autor is wangliang1986